src/imars3d/backend/autoredux.py

```diff
@@ -23,7 +24,7 @@ def auto_reduction_ready(data_file: str) -> bool:
     raise NotImplementedError
 
 
-def load_template_config() -> dict:
+def load_template_config(config_path) -> dict:
     """
     Load the template configuration file.
 
```

Comment:
This docstring doesn't follow the [Numpy docstring style](https://numpydoc.readthedocs.io/en/latest/format.html) we're following.
You need to add info for sections [Parameters](https://numpydoc.readthedocs.io/en/latest/format.html#parameters), [Returns](https://numpydoc.readthedocs.io/en/latest/format.html#returns) and [Raises](https://numpydoc.readthedocs.io/en/latest/format.html#raises)